<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayer</class>
 <widget class="QWidget" name="MediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#VideoContainer {
	background-color: black;
}

QPushButton {
	border: 0px;
}

QPushButton:pressed {
	border: 0px;
}

QFrame#controlsFrame {
	border: 0px;
	border-top: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0.00568182 rgba(198, 198, 198, 255), stop:1 rgba(169, 169, 169, 255));
}

QLabel#timeLabel {
	color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0.00568182 rgba(198, 198, 198, 255), stop:1 rgba(169, 169, 169, 255));
}

QPushButton#stopButton {
	border-left: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0.00568182 rgba(198, 198, 198, 255), stop:1 rgba(169, 169, 169, 255));
	border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0.00568182 rgba(198, 198, 198, 255), stop:1 rgba(169, 169, 169, 255));
}

QSlider::handle:horizontal {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #eee, stop:1 #ccc);
	border: 1px solid #777;
	width: 13px;
	margin-top: -2px;
	margin-bottom: -2px;
	border-radius: 4px;
}

QSlider::handle:horizontal:hover {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1,stop:0 #fff, stop:1 #ddd);
	border: 1px solid #444;
	border-radius: 4px;
}

QSlider::handle:horizontal:disabled {
	background: #eee;
	border: 1px solid #aaa;
	border-radius: 4px;
}

QPushButton#playButton { background-image: url(:/images/start.png); background-position: center center; }
QPushButton#playButton:hover { background-image: url(:/images/start-hover.png); }
QPushButton#playButton:pressed {  background-image: url(:/images/start-pressed.png); }

QPushButton#pauseButton { background-image: url(:/images/pause.png); background-position: center center; }
QPushButton#pauseButton:hover { background-image: url(:/images/pause-hover.png); }
QPushButton#pauseButton:pressed {  background-image: url(:/images/pause-pressed.png); }

QPushButton#stopButton { background-image: url(:/images/stop.png); background-position: center center; }
QPushButton#stopButton:hover { background-image: url(:/images/stop-hover.png); }
QPushButton#stopButton:pressed {  background-image: url(:/images/stop-pressed.png); }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="controlsFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pauseButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>22</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeLabel">
          <property name="text">
           <string>0 / 0</string>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="timeSlider">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>14</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
